src/main/java/graphql/validation/constraints/standard/PatternConstraint.java

```diff
@@ -51,40 +44,28 @@ public boolean appliesToType(GraphQLInputType inputType) {
     @Override
     protected List<GraphQLError> runConstraint(ValidationEnvironment validationEnvironment) {
         Object validatedValue = validationEnvironment.getValidatedValue();
-        GraphQLInputType argumentType = validationEnvironment.getValidatedType();
 
-        if (validatedValue == null) {
-            return emptyList();
-        }
```

Comment:
Another another  of null being allowed

I dont recall exactly of hand out null handling policy but I am pretty sure we want graphql to catch null / non null lissues and these constraints o catch the rest